src/createStore.js

```diff
@@ -18,10 +39,23 @@ export default function createStore(reducer, initialState) {
   var currentState = initialState;
   var listeners = [];
 
+  /**
+   * Reads the state tree managed by the store.
+   *
+   * @returns {any} The current state tree of your application.
+   */
   function getState() {
     return currentState;
   }
 
+  /**
+   * Adds a change listener. It will be called any time an action is dispatched,
+   * and some part of the state tree may potentially have changed. You may then
+   * call `getState()` to read the current state tree inside the callback.
+   *
+   * @param {Function} listener A callback to be invoked on every dispatch.
```

Comment:
A good way to document callbacks is to use the @callback tag. This way you can document the @params of the callback function.